workers/exclusive-content/src/lib/requestHandlers.ts

```diff
@@ -9,14 +9,16 @@ import {
 } from './helpers'
 import { verifyReceipt } from './receiptVerifier'
 
+headers.set('Content-Type', 'application/json;charset=UTF-8')
```

Comment:
mutating this global is a little weird. Is this change meant to affect the object everywhere?